src/shared/journal-file-util.c

```diff
@@ -318,12 +320,25 @@ int journal_file_set_offline(JournalFile *f, bool wait) {
 
         target_state = f->archive ? STATE_ARCHIVED : STATE_OFFLINE;
 
+        log_ratelimit_full(LOG_DEBUG,
+                           JOURNAL_LOG_RATELIMIT,
+                           "Journal file %s is %ssynchronously transitioning to %s.",
+                           f->path,
+                           !wait ? "a" : "",
```

Comment:
Please make the ternary op provide a word, rather than prefix.
E.g. we usually do
```
"... %s ...", b ? "enable" : "disable"
```
rather than
```
"... %sable ...", b ? "en" : "dis"
```